recipes-graphics/wayland/wayland_1.16.%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append += " file://fix_duplicate_main.patch "
```

Comment:
Usually we do not use `+=` and `_append` together, its better to use
`SRC_URI_append = " file://fix_duplicate_main.patch "`